### ISSUE TITLE
Feature: Archive Repo

### ISSUE THREAD

**Issue by daank-c** [dogecoin/dogecoin.com#340]
This isn't so much of a "feature", but I was thinking that it might be good to archive this repository if it is no longer being used/maintained for dogecoin.com. I say this because I submitted a PR earlier #339, but have been made aware that this was the wrong repo.

It might also be good to write a line at the top of `README.md` at the root of this project that points people to the new repo. If that's of interest to anyone, I can submit a PR and add that.

**Comment by AbhayJha24** [comment]
Yes please do it, I want to contribute to the project but since this repository is no longer maintained, I am unable to contribute to the project.
